tools/perform_upgrade.sh

```diff
@@ -20,10 +20,10 @@
 
 set -e
 
-BOARD="$1"
+BOARD="${1}"
 
-./deploy.py --board=$BOARD --opensk --programmer=none
-python3 -m tools.deploy_partition --board=$BOARD "$2"
+./deploy.py --board=$BOARD --opensk --programmer=none $2
```

Comment:
nit: Would be nice to also fix the `"${BOARD}"` here too :-)

Reply:
Oops, forgot that one.